packcheck.sh

```diff
@@ -1693,10 +1627,8 @@ test -n "$1" \
     || { short_help; echo -e "\nTry --help for detailed help"; exit 1; }
 
 case $1 in
-  cabal) shift; eval_env "$@"; BUILD=cabal-v1; \
-    echo "DEPRECATED! please use cabal-v1 instead";;
-  cabal-v1) shift; eval_env "$@"; BUILD=cabal-v1; \
-    echo "DEPRECATED! please use cabal-v2 instead";;
+  cabal) die "DEPRECATED! please use cabal-v2 instead";;
+  cabal-v1) die "cabal-v1 is not supported, please use cabal-v2 instead";;
```

Comment:
`die` here because I don't want to change the behavior, nor do I want to proceed after this point.